### DISCUSSION TITLE
Is it possible to have +/- at the beginning of the number input?

### DISCUSSION THREAD

**Discussion by rikiless** [uNmAnNeR/imaskjs#845] [General]
Want to achieve values such as `+00.00`, `+1.50`, `-0.50`, etc,.. Is it possible?

**Comment by uNmAnNeR** [comment]
hi @rikiless,
for minus it seems fine already, right?
for plus do you want it to be added automatically for positive numbers or should be manually entered?

**Reply by rikiless** [reply]
It would be better if it "+" will be added automatically.
